config/config.yaml

```diff
@@ -0,0 +1,78 @@
+# netbox device attributes that should be used as filters for devices query.
+# these translate directly into the return fields from the netbox json API.
+# If no filter is specified, by default only devices with Active (1) status are pulled.
+filters:
+  status: ["active", "staged", "offline"]
+#  manufacturer: ['juniper']
+#  custom_fields.field1: "foo"
+
+# list of tags to filter devices in Netbox. Only devices that have this tag will be 
+# synchronized to NetSpyGlass. Default is None, which causes all devices present in Netbox to be synchronized.
+whitelist:
+#  - tag1
+#  - tag2
+
+# list of tags to blacklist from netbox. Devices that have this tag will not be synchronized to NetSpyGlass.
+# Blacklist filter is applied after the whitelist (if any). Default is None, which turns this off and causes all
+# devices that pass whitelist will be synchronized
+blacklist:
+#  - tagq
+
+# list of devices in nsg excluding from synchronization 
+nsg_blacklist:
+  name:
+#    - nsg device name
+  deviceId:
+#    - nsg device id
+  address:
+#    - nsg device ip
+
+# Mapping device name -> snmp channels. Device name can be in regex format
+default_channels:
+#  device_name:
+#    - v1public
+#  aaa.*|bbb-bb01|ccc-mt01:  # device names regex
+#    - v1public
+  "*":
+    - v2public
+
+#domain: your.domain.com
+
+# List of extracting external nsg tags with corresponding netbox attribute from netbox devices API
+# format:
+#  nsg_tag:  - name of tag saved to nsg db
+#    path:   - path to the device property within netbox device model in dot notation
+#    transform:  - how to transform netbox attribute to nsg tag: upper, lower, capitalize
+#    mapping:    - dict to convert multiple netbox attribute value to single nsg tag
```

Comment:
this comment is a good start but it is not enough. I do not undersrand how does "mapping" work. Perhaps you can provide some examples? Alternatively, add a section to README to explain this in more details with examples